### PR TITLE
Move README content to contribution section.

### PR DESCRIPTION
**Pull request by steveperry-53**
Move content from README and editdocs into the new contribution section.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1861)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,182 +1,13 @@
...
-  [best practices.](http://kubernetes.io/docs/user-guide/config-best-practices/)
-
-Don't assume the reader has this repository checked out, use `kubectl
-create -f https://github...` in example commands. For Docker images
-used in config yamls, try to use an image from an existing Kubernetes
-example. If creating an image for a doc, follow the
-[example guidelines](https://github.com/kubernetes/kubernetes/blob/master/examples/guidelines.md#throughout)
-section on "Docker images" from the Kubernetes repository.
-
-## Partners
```
Comment (jaredbhatti) [inline README.md]: The partners content should live somewhere.  Perhaps adding this section to the bottom of the partners page?

File: README.md [hunk 2]
```diff
@@ -1,182 +1,13 @@
...
-Kubernetes partners refers to the companies who contribute to the Kubernetes core codebase, extend their platform to support Kubernetes or provide managed services to users centered around the Kubernetes platform. Partners can get their services and offerings added to the [partner page](https://k8s.io/partners) by completing and submitting the [partner request form](https://goo.gl/qcSnZF). Once the information and assets are verified, the partner product/services will be listed in the partner page. This would typically take 7-10 days.
-
-## Thank you!
```
Comment (jaredbhatti) [inline README.md]: I kind of like leaving the "Thank You" section in.

File: README.md [hunk 3]
```diff
@@ -1,182 +1,13 @@
...
-YAML,yaml,service-sample.yaml,/docs/user-guide/services/service-sample.yaml&#123;% endcapture %&#125;
-&#123;% include tabs.html %&#125;</pre>
-
-In English, this would read: "Create a set of tabs with the alias `servicesample`,
-and have tabs visually labeled "JSON" and "YAML" that use `json` and `yaml` Rouge syntax highlighting, which display the contents of
-`service-sample.{extension}` on the page, and link to the file in GitHub at (full path)."
-
-Example file: [Pods: Multi-Container](http://kubernetes.io/docs/user-guide/pods/multi-container/).
-
-## Use a global variable
```
Comment (jaredbhatti) [inline README.md]: I don't see "Global variable" content listed anywhere else. Shouldn't this move somewhere?

Comment (jaredbhatti) [inline README.md]: I'd suggest "Write a new topic"

**Comment by jaredbhatti** [comment]
Added a few comments.

File: README.md (hunk 3 above)
Comment (steveperry-53) [inline README.md]: Let's discuss. I'd have to understand these global variables better to know whether they still apply.

Comment (steveperry-53) [inline README.md]: @devin-donnelly, @jaredbhatti, and I discussed this. We decided to omit the bit about global variables from the Kubernetes docs, for now.

File: partners/index.html [hunk 4]
```diff
@@ -14,7 +14,11 @@
 
 <section id="users">
 	<main>
+<<<<<<< HEAD
```
Comment (janetkuo) [inline partners/index.html]: Unresolved merge conflict here

**Pull request merged**